src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { FC, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './Types/Types';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function findUser(userId: number) {
+  return usersFromServer.find(({ id }) => id === userId);
+}
+
+function getTodos(todos: Todo[]) {
+  return todos.map(todo => {
+    const cloneTodo = { ...todo };
+
+    cloneTodo.user = findUser(cloneTodo.userId);
+
+    return cloneTodo;
+  });
+}
+
+export const App: FC = () => {
+  const visibleTodo = getTodos(todosFromServer);
+
+  const [todos, setTodos] = useState(visibleTodo);
+
+  const [selectUserId, setSelectUserId] = useState(0);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [todoForList, setTodoForList] = useState('');
+
+  const [validTitle, setValidTitle] = useState(false);
+  const [validSelect, setValidSelect] = useState(false);
```

Comment:
add a question prefix whenever you use boolean as a value
```suggestion
  const [isValidTitle, setIsValidTitle] = useState(false);
  const [isValidSelect, setIsValidSelect] = useState(false);
```